packages/cover-traffic/src/state.ts

```diff
@@ -17,17 +19,20 @@ export type OpenChannels = {
 
 export type State = {
   nodes: Record<string, PeerData>
+  // channels indexed by its channelId
   channels: Record<string, ChannelData>
   log: string[]
+  // currently non-closed cover traffic channels
   ctChannels: OpenChannels[]
   block: BN
+  // number of failed messages indexed by the base58-encoded string of node id.
   messageFails: Record<string, number>
 }
 
 export type PeerData = {
-  id: any //PeerId,
+  id: any //PeerId type, as implemented in IPFS
```

Comment:
Should this not be properly typed then?

Reply:
I think it should work. Want to double check with @peterbraden for any other consideration that I overlooked.